src/masonite/snippets/auth/templates/auth/login.html

```diff
@@ -32,4 +46,4 @@ <h5 class="mb-0">Login</h5>
         </div>
     </div>
 </div>
-{% endblock %}
+{% endblock %}
```

Comment:
An empty line in the missing of the file is missing.

Reply:
Done !